src/app/drivingLicence/controllers/root.js

```diff
@@ -1,19 +1,73 @@
 const { Controller: BaseController } = require("hmpo-form-wizard");
 
+const {
+  API: {
+    BASE_URL,
+    PATHS: { PERSON_INFO }
+  }
+} = require("../../../lib/config");
+
 class RootController extends BaseController {
   async saveValues(req, res, next) {
-    const sharedClaims = req.session?.shared_claims;
+    req.sessionModel.reset();
+    req.sessionModel.set("isAuthSourceRoute", false);
 
-    if (sharedClaims) {
-      if (sharedClaims?.names?.length > 0) {
-        req.journeyModel.set("surname", sharedClaims.names[0]?.familyName);
-        req.journeyModel.set("givenNames", sharedClaims.names[0]?.givenNames);
-      }
+    let personInfoApiResponse = {};
 
-      if (sharedClaims?.dateOfBirths?.length > 0) {
-        req.journeyModel.set("dateOfBirth", sharedClaims.dateOfBirths[0]);
+    const headers = {
+      session_id: req.session?.tokenId
+    };
+
+    if (process.env.AUTH_SOURCE_ENABLED === "true") {
```

Comment:
Should this be isAuthSourceRoute
if (process.env.AUTH_SOURCE_ENABLED && !isAuthSourceRoute)

That way we only run it once per journey